release/RELEASE.md

```diff
@@ -1,124 +1,142 @@
 
-## Making a Snapshot Release (1.x)
+The process is similar to the weekly snapshot, except that both the [daml] and
+[canton] bundles have to be manually triggered. Specifically:
 
-At this point, snapshot releases on 1.x should be rare, and only meant to be
-precursors to stable, patch releases. First, identify which release branch
-you're patching, and what the next version number should be. In this example
-we'll assume you're patching the 1.18 branch to produce a snapshot (likely a
-release candidate) for 1.18.3.
 
+0. Make sure all the changes you want are in the release branch.
+1. Start _from latest `main`_ and run
+   ```
+   $ ./release.sh snapshot origin/release/2.0.x 2.0.1
+   ```
+   The output will be a line that starts with a commit sha, followed by a
+   snapshot release number. You need to take that line and add it to the [ LATEST`]
+   file, adding ` SPLIT_RELEASE` at the end of that line. You should put that line
+   in the file so as to preserve semver ordering, and overwrite any existing
+   snapshot with the same prefix.
+2. Make a PR against the `main` branch with just that one line added, touching
+   no other file. Add the `Standard-Change` label to that PR.
+3. When the PR is merged, the build of the corresponding commit on `main` will
+   create a "split release" bundle and push it to Artifactory. It should notify
+   on `#ci-failures-daml` on Slack.
+4. The next step is to request a build from the [Canton] team (on
+   `#team-canton`) that relies on the RC snapshot. Once you have a [canton]
+   release, you can proceed.
+5. Go to the [assembly] repo, and follow the instructions there to make a release
+   using the [Canton] version that was just created. The `LATEST` file on the
+   [assembly] repo only contains on version; it is safe to overwrite it, and to
+   "go backwards" if needed.
+6. Once the `main` build of the [assembly] repo has finished, you should
+   proceed with testing. You should open up this document _in the branch of
+   the release you're making_, as testing instructions change over time.
+7. After testing, if everything went well, you should go on to turning the RC
+   into a stable release. If something went wrong, make appropriate changes to
+   the release branch and start over.
+
+## Stable 2.x
+
+Making a stable release follows the same steps as a snapshot RC, except that:
+
+- You should not be choosing an arbitrary commit, you should pick the latest RC
+  for the branch.
+- Go through the [checklist] before making the release. Note that
+  the checklist is not available publicly.
+- Instead of adding a line to [`LATEST`], remove the `-snapshot` part of the
+  version number for the existing RC.
+- Similarly, modifying the `LATEST` file on the assembly repo _should_ amount
+  to just removing the `-` parts of the version numbers for both Canton and
+  daml, but follow the instructions there.
+- You need a team lead to approve the PR on both the [daml] and [assembly] repos.
```

Comment:
Do we need to change the GitHub release from pre-release to release for stable?

Reply:
Yes. I thought I'd put that in. Either I accidentally deleted that in a later edit round, or I'm losing my mind.

I'll address it.